appveyor.yml

```diff
@@ -4,17 +4,14 @@
 # https://github.com/rmcgibbo/python-appveyor-conda-example
 
 
-matrix:
-  fast_finish: true     # immediately finish build once one of the jobs fails.
```

Comment:
@jorisvandenbossche I removed this b/c we still want to other jobs to finish even when the first one did not.